Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/TwoHandManipulatable.cs

```diff
@@ -132,9 +106,9 @@ private bool ShowBoundingBox
         /// <summary>
         /// SetManipulationMode
         /// </summary>
-        private void SetManipulationMode(TwoHandedManipulation mode)
+        public void SetManipulationMode(ManipulationMode mode)
```

Comment:
Shouldn't this be a property?

Also, there is no GetManipuationMode… client code should be able to see what the current value is as well as set a new one.

Reply:
That's addressed in https://github.com/cre8ivepark/MixedRealityToolkit-Unity/pull/2

Reply:
I made it a property and marked this method obsolete.